src/Controller/Backend/BackendBase.php

```diff
@@ -71,6 +71,7 @@ public function before(Request $request, Application $app, $roleRoute = null)
         // Sanity checks for doubles in in contenttypes. This has to be done
         // here, because the 'translator' classes need to be initialised.
         $app['config']->checkConfig();
+        $app['config']->checkTaxonomy();
```

Comment:
Also moving the call means we can make that function private and not expose another API